pkg/assessor/manifest/manifest.go

```diff
@@ -210,6 +227,38 @@ func useADDstatement(cmdSlices map[int][]string) bool {
 	return false
 }
 
+func sensitiveVars(cmd string) bool {
+	if !strings.Contains(cmd, "=") {
+		return false
+	}
+
+	for _, s := range sensitiveWords {
+		sensitiveWords = append(sensitiveWords, strings.ToUpper(s))
+	}
+	pat := strings.ReplaceAll(`.*(REP).*`, "REP", strings.Join(sensitiveWords, "|"))
+	r, _ := regexp.Compile(pat)
```

Comment:
Don't ignore `regexp.Compiler` error, it could failed from a malformed regex originated from user input.